__tests__/app.test.js

```diff
@@ -64,6 +59,43 @@ describe("4. GET /api/reviews", () => {
       });
   });
 });
+describe("GET /api/reviews/:review_id", () => {
+  test("GET 200 - responds with a review object with the correct properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
```

Comment:
Can change toEqual to a more flexible matcher e.g. expect.objectContaining or .toMatchObject